controllers/network/ipset_controller.go

```diff
@@ -43,6 +45,8 @@ import (
 	util "github.com/openstack-k8s-operators/lib-common/modules/common/util"
 )
 
+const CtlPlaneNetwork = "ctlplane"
```

Comment:
nit: I don't think this needs to be Heap allocated right? Looks like we're only using this const in the `reconcileNormal` function, so the const can be stack allocated in that function.

Unless you were planning to use it externally somewhere.

Reply:
`const` is a compile time construct in go, so there is no question heap/stack memory allocation for them at run time. They're logically replaced in code at compile time.